CHANGELOG.md

```diff
@@ -1,3 +1,56 @@
+# 1.7.0-beta01 (August 2024)
+
+_Changes since release/1.7.0-alpha03_
+
+## Breaking changes
+### iOS
+- [`UIKitView` and `UIKitViewController` in `package androidx.compose.ui.interop` are deprecated. New API are mentioned in deprecation message. Deprecated invocations should work fine unless custom `onResize` is used, it is disallowed now and will print a warning.](https://github.com/JetBrains/compose-multiplatform-core/pull/1494)
+- [Actual of expected `InteropView` on iOS is `UIResponder` now instead of `UIView`. It's the first common ancestor for `UIViewController` and `UIView`, both of which can be integrated using iOS interop APIs](https://github.com/JetBrains/compose-multiplatform-core/pull/1489)
+- [The app will crash by default, if `CADisableMinimumFrameDurationOnPhone` is not set to true in `Info.plist`. Use newly added `ComposeUIViewControllerConfiguration.enforceStrictPlistSanityCheck` to opt-out of this behavior](https://github.com/JetBrains/compose-multiplatform-core/pull/1451)
+
+## Features
+### Multiple Platforms
+- [New multiplatform module "material-navigation"](https://github.com/JetBrains/compose-multiplatform-core/pull/1504)
+
+### iOS
+- [New `UIKitView` and `UIKitViewController` API in `package androidx.compose.ui.viewinterop`. Support of `onReset` to reuse the interop composable emitted node and avoid excessive native views reallocations, fine-grain touches strategy control (cooperative with explicit time delay, non-cooperative where no touches are received by Compose, ignoring touches)](https://github.com/JetBrains/compose-multiplatform-core/pull/1494)
+
+### Resources
+- [Added support of test resources in Compose Multiplatform projects](https://github.com/JetBrains/compose-multiplatform/pull/5122)
+- [Added support of multi-module resources in JVM-only projects](https://github.com/JetBrains/compose-multiplatform/pull/5122)
+
+## Fixes
+### Multiple Platforms
+- [_(prerelease fix)_ Fix redirect on android artifacts for "window-core" module](https://github.com/JetBrains/compose-multiplatform-core/pull/1506)
```

Comment:
Multiple Platforms -> Android?

Reply:
Asked @terrakok, he said that it affected all platforms before that fix, even without an android target. I guess it should be as it is